frontend/src/metabase/nav/containers/Navbar.jsx

```diff
@@ -10,20 +9,19 @@ import { push } from "react-router-redux";
 import cx from "classnames";
 import { t } from "ttag";
 import { Flex, Box } from "grid-styled";
-import styled from "styled-components";
-import { space } from "styled-system";
 
 import * as Urls from "metabase/lib/urls";
-import { color, darken, lighten } from "metabase/lib/colors";
+import { color, darken } from "metabase/lib/colors";
 
 import Icon, { IconWrapper } from "metabase/components/Icon";
 import EntityMenu from "metabase/components/EntityMenu";
 import Link from "metabase/components/Link";
 import LogoIcon from "metabase/components/LogoIcon";
-import OnClickOutsideWrapper from "metabase/components/OnClickOutsideWrapper";
 import Modal from "metabase/components/Modal";
 
 import ProfileLink from "metabase/nav/components/ProfileLink";
+import SearchBar from "metabase/nav/containers/SearchBar";
```

Comment:
The container/component distinction has fallen apart a bit, but I think should be in "metabase/nav/components".

Reply:
Yeah in this PR that makes sense. I had added it to `containers` in anticipation of #14229 since that adds data loading to the search bar but since it may take a moment for that to land I'll add it to `components` instead.